### DISCUSSION TITLE
Modal close modal

### DISCUSSION THREAD

**Discussion by reinisg** [incluud/accessible-astro-components#41] [Q&A]
Is it possible to change modal close button icon to custom icon? If it is possible, could some please provide example? 

Regards, Reinis

**Comment by markteekman** [comment]
Hey @reinisg!

Currently it's only possible to set a text string for the close button. You could work around it though with something like this:

```js
<script>
  const modalCloseButtons = document.querySelectorAll('.modal__close button')
  modalCloseButtons.forEach((button) => {
    button.innerHTML = `
      <span class="sr-only">Close</span>
      <svg width="20" height="20" aria-hidden="true" fill="black" xmlns="http://www.w3.org/2000/svg"><path d="M10 10 2 2m8 8 8 8m-8-8 8-8m-8 8-8 8" stroke="black" stroke-width="2.667" stroke-linecap="round" stroke-linejoin="round"/></svg>
    `
  })
</script>
```

```css
.modal__close button {
  display: flex;
  justify-content: flex-end;
}
```

Which would give you the following result:

<img width="706" alt="modal-example" src="https://user-images.githubusercontent.com/3909046/230652056-fe5c4372-f326-4a98-86bd-44adcbd85dd6.png">
